Source/Core/DolphinQt/TAS/GCTASInputWindow.cpp

```diff
@@ -84,6 +77,40 @@ GCTASInputWindow::GCTASInputWindow(QWidget* parent, int num) : TASInputWindow(pa
   setLayout(layout);
 }
 
+void GCTASInputWindow::SetTriggersBox()
+{
+  m_triggers_box = new QGroupBox(tr("Triggers"));
+
+  const QKeySequence l_shortcut_key_sequence = QKeySequence(Qt::ALT + Qt::Key_N);
+  const QKeySequence r_shortcut_key_sequence = QKeySequence(Qt::ALT + Qt::Key_M);
+
+  auto* l_label = new QLabel(QStringLiteral("%1 (%2)").arg(
```

Comment:
```suggestion
  auto* l_label = new QLabel(tr("Left (%1)").arg(l_shortcut_key_sequence.toString(QKeySequence::NativeText));
```

You generally want to allow translators full control over the string, since not every language follows left-to-right reading (or even the same conventions as English).

Ditto for the string related to the right label below.